apps/dashboard/pkg/migration/conversion/testdata/output/v1beta1.panel-conversions.v2alpha1.json

```diff
@@ -0,0 +1,386 @@
+            }
+          ],
+          "data": {
+            "kind": "QueryGroup",
+            "spec": {
+              "queries": [
+                {
+                  "kind": "PanelQuery",
+                  "spec": {
+                    "query": {
+                      "kind": "DataQuery",
+                      "spec": {
+                        "bucketAggs": [
+                          {
+                            "field": "@timestamp",
+                            "id": "2",
+                            "type": "date_histogram"
+                          }
+                        ],
+                        "query": "*"
+                      }
+                    },
+                    "datasource": {
+                      "type": "elasticsearch",
+                      "uid": "existing-target-uid"
+                    },
+                    "refId": "A",
+                    "hidden": false
+                  }
+                }
+              ],
+              "transformations": [
+                {
+                  "kind": "groupBy",
+                  "spec": {
+                    "id": "groupBy",
+                    "options": {
+                      "fields": {
+                        "status": {
+                          "aggregations": [
+                            "count"
+                          ],
+                          "operation": "groupby"
+                        }
+                      }
+                    }
+                  }
+                },
+                {
+                  "kind": "sortBy",
+                  "spec": {
+                    "id": "sortBy",
+                    "options": {
+                      "sort": [
+                        {
+                          "desc": true,
+                          "field": "count"
+                        }
+                      ]
+                    }
+                  }
+                }
+              ],
+              "queryOptions": {
+                "timeFrom": "1h",
+                "maxDataPoints": 1000,
+                "timeShift": "1h",
+                "queryCachingTTL": 300,
+                "hideTimeOverride": true
+              }
+            }
+          },
+          "vizConfig": {
+            "kind": "barchart",
+            "spec": {
+              "pluginVersion": "9.5.0",
+              "options": {
+                "barWidth": 0.97,
+                "groupWidth": 0.7,
+                "orientation": "horizontal"
+              },
+              "fieldConfig": {
+                "defaults": {
+                  "color": {
+                    "mode": "continuous-GrYlRd"
+                  },
+                  "custom": {
+                    "orientation": "horizontal"
+                  }
+                },
+                "overrides": []
+              }
+            }
+          }
+        }
+      }
+    },
+    "layout": {
+      "kind": "GridLayout",
+      "spec": {
+        "items": null
```

Comment:
This is not correct. The grid layout should have contained the grid items that reference the panels.

The layout is being overwritten here: https://github.com/grafana/grafana/blob/305ed25896bcc4530aa23138e407a786d9e8f602/apps/dashboard/pkg/migration/conversion/v1.go#L40-L45


It's not in this PR but something that was left from before.

Reply:
fixed this in [#1479cbf](https://github.com/grafana/grafana/pull/109037/commits/1479cbf80ae6fe2ecf48cabf8f85246bffc2afd2) so I can get on with merging the library panel changes.